recipes/gstreamer/build.sh

```diff
@@ -8,6 +8,7 @@ cp -r $PREFIX/lib/libffi* $PREFIX/lib64
 # be included in the package (it is about 12MB).
 ./configure --disable-examples --prefix="$PREFIX" --datarootdir=`pwd`/tmpshare
```

Comment:
A `data` directory in Linux us usually $PREFIX/share but now I see that is not what I was thinking... It looks like it is some data bundled in the source instead.